OpenCore-Boot.sh

```diff
@@ -13,6 +13,10 @@
 # NOTE: Tweak the "MY_OPTIONS" line in case you are having booting problems!
 ############################################################################
 
+# Tell git to ignore changes in files that will change anyway after first run
+# https://compiledsuccessfully.dev/git-skip-worktree/
+git update-index --skip-worktree OVMF_VARS*.fd
```

Comment:
Now I'm not sure about this anymore, it seems smart, but you also want to be able to go back to the templated file easily. Maybe the originals should be stored in a folder somewhere, and copied over to the main folder the first time?

Reply:
> Maybe the originals should be stored in a folder somewhere, and copied over to the main folder the first time?

This seems like a nice approach. Copy if not exists, and have where its copied to be ignored by git. Great!